src/Domain/Session/SessionService.ts

```diff
@@ -1,12 +1,17 @@
 import * as crypto from 'crypto'
 import * as winston from 'winston'
+import * as moment from 'moment'
```

Comment:
Moment seems to be deprecated: https://github.com/moment/moment/

> Moment.js is a legacy project, now in maintenance mode. In most cases, you should choose a different library.

Reply:
oh wow .. I feel so old :'D will replace